apps/emqx/src/emqx_cm.erl

```diff
@@ -80,9 +80,15 @@
         , mark_channel_connected/1
         , mark_channel_disconnected/1
         , get_connected_client_count/0
+
```

Comment:
Extra/empty line for a reason?

Reply:
Yes, to logically separate a group of functions.